### PR TITLE
Update readme and release docs

### PR DESCRIPTION
**Pull request by skarimo**
SSIA

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,6 +110,29 @@ The following Datadog resources can be registered within your AWS account. Refer
 | User                    | `Datadog::IAM::User`              | [ Create and manage Datadog users][13]                  | `datadog-iam-user`              | [Schema Handler Versions][14] |
 | SLOs                    | `Datadog::SLOs::SLO`              | [ Create and manage Datadog SLOs][19]                   | `datadog-slos-slo`              | [Schema Handler Versions][20] |
 
+## Supported Regions
```
Comment (ruthnaebeck) [inline README.md:113]:
```suggestion
## Supported regions
```

File: README.md [hunk 2]
```diff
@@ -114,6 +110,29 @@ The following Datadog resources can be registered within your AWS account. Refer
...
+- us-west-2
+- ap-northeast-2
+- ap-south-1
+- ap-southeast-1
+- us-east-1
+- eu-west-1
+- ap-northeast-1
+- us-east-2
+
+*Note*: To privately register a resource in any other region, use the provided packages.
```
Comment (ruthnaebeck) [inline README.md:134]:
```suggestion
**Note**: To privately register a resource in any other region, use the provided packages.
```

File: RELEASING.md [hunk 3]
```diff
@@ -36,8 +36,16 @@ Our team will trigger the release pipeline.
...
-* Run the `publish` script `./publish -f <RESOURCE_DIRECTORY> -r <REGION> -t <RESOURCE_TYPE_NAME> -v <VERSION>`.
 * Upload the generated ZIP file to the s3 bucket `aws s3 cp <ZIP_FILE> s3://datadog-cloudformation-resources/<RESOURCE_NAME>/<RESOURCE_NAME>-<VERSION>.zip`.
+* Run
+    ```
+    aws cloudformation update-stack-set --stack-set-name <RESOURCE_NAME> \
+      --use-previous-template \
+      --parameters '[{"ParameterKey": "VersionToPublish", "ParameterValue":"<VERSION_TO_DEPLOY>"}]' \
+      --operation-preferences '{"RegionConcurrencyType": "<SEQUENTIAL|PARALLEL>", "FailureToleranceCount": "16"}'
+    ```
+* Check the StackSet in AWS to ensure deploys in all regions succeed
```
Comment (ruthnaebeck) [inline RELEASING.md:50]:
```suggestion
* Check the StackSet in AWS to ensure deploys in all regions succeed.
```

**Review by ruthnaebeck — CHANGES_REQUESTED** [review]
Docs review

File: RELEASING.md [hunk 4]
```diff
@@ -36,8 +36,16 @@ Our team will trigger the release pipeline.
 * Bump the version in `version.py`.
 * Bump the version in the `description` field of the resource's JSON schema.
 * Run `cfn submit --dry-run` to generate a zip file of the resource.
+* Run `cfn generate` code based on the project and resource type schema.
```
Comment (zippolyte) [inline RELEASING.md:39]: Should be done before submit probably

**Review by ruthnaebeck — APPROVED** [review]
👍 for docs

**Review by zippolyte — APPROVED** [review]

**Pull request merged**